src/sdk/nsfs_glacier_backend/helper.js

```diff
@@ -0,0 +1,29 @@
+/* Copyright (C) 2024 NooBaa */
+'use strict';
+
+/**
+ * This module exists so as to export the common function `getGlacierBackend`
+ * 
+ * Keeping this in the generic.js creates cyclic dependency issue.w
+ */
+
+const config = require('../../../config');
+const { TapeCloudGlacierBackend } = require('./tapecloud');
+// eslint-disable-next-line no-unused-vars
+const { GlacierBackend } = require('./backend');
```

Comment:
still needed?

Reply:
Actually that's just for JSDoc so that we can have autocompletes working properly.